Composer/packages/lib/shared/src/lgUtils/stringBuilders/buildLgType.ts

```diff
@@ -13,7 +13,7 @@ import LgType from '../models/LgType';
  */
 export default function buildLgType(lgType: LgType): string {
   const { hostKind, hostField } = lgType;
-  const kindPrefix = hostKind.replace('Microsoft.', '');
+  const kindPrefix = hostKind.replace('Microsoft.', '').replace('.', '');
```

Comment:
This seems like an odd change to me. Can you give me an example of where we'd need to get rid of more periods?

Reply:
Should this be `.replace('Teams.','');` ?

Reply:
@benbrown I escaped the `.` here because $kind is used to generate lg template name, however, lg utils can only recongize \w+ pattern, so there is a trailing replace statement to remove all dots.

@EricDahlvang I didn't write `replace('Teams.')` here is because of Teams namespace logic shouldn't be exposed to Composer domain. We can remove 'Microsoft.' suffix because we know all builtin $kinds are in the `Microsoft.` namespace, but for the namespace `Teams.`, I consider it as an action extension.